src/helpers/cli/mod.rs

```diff
@@ -36,17 +41,22 @@ impl ShellText {
     }
 }
 
-pub fn as_code_lines(code: &str, start_line: u32) -> Vec<String> {
+pub fn as_code_lines(code: &str, start_line: usize) -> Vec<String> {
     code.lines()
         .map(|line| {
-            ShellText::CodeColor(Box::new(ShellText::Literal(line.to_string()))).to_string()
+            format!(
+                "{}{}{}",
+                color::Fg(color::LightBlue),
+                line,
+                color::Fg(color::Reset)
+            )
         })
         .enumerate()
         .map(|(i, line)| {
-            let line_num = ShellText::Dim(Box::new(ShellText::Literal(
-                format!("{}    ", i as u32 + start_line)[..4].to_string(),
-            )));
-            format!("{} {}", line_num.to_string(), line)
+            let mut line_num = format!("{}{}    ", style::Faint, i + start_line);
+            line_num.truncate(6);
```

Comment:
This looks suspicious to me. What are you trying to do?

Reply:
I'd expect rust to have width-sensitive format specifiers, like `printf` does.

Reply:
Updated to use format specifier